String_Exercises.js

```diff
@@ -0,0 +1,185 @@
+//1. `console.log` the first character of a string.
+console.log(' ');
+console.log('Question 1~~~~~~~~~');
+console.log(' ');
+
+let name = 'Kameron';
+
+console.log('The first letter of my name is ' + name[0] + '.');
+
+//2. `console.log` the length of a string.
+console.log(' ');
+console.log('Question 2~~~~~~~~~');
+console.log(' ');
+
+let company = 'From Software';
+
+console.log('My favorite company has ' + company.length + ' characters.');
+
+//3. `console.log` the last character of any string.
+console.log(' ');
+console.log('Question 3~~~~~~~~~');
+console.log(' ');
+
+let food = 'Apple';
+
+console.log('The last letter in ' + food + ' is ' + food[food.length - 1] + '.');
+
+//4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+console.log(' ');
+console.log('Question 4~~~~~~~~~');
+console.log(' ');
+
+let pet = 'dinosaur';
+
+let copyPet = pet.slice(0, pet.length - 1) + pet[pet.length - 1].toUpperCase();
+
+console.log('Jurassic Park is now open with our new ' + copyPet + '.');
+
+//5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+//your code block should add '(pinky)' at the end of the amount if it's 1 million.
+console.log(' ');
+console.log('Question 5~~~~~~~~~');
+console.log(' ');
+
+for (amount = 10; amount <= 1000000; amount *= 10) {
+  if (amount === 1000000) {
+    console.log(amount + ' dollars (pinky).');
+  } else {
+    console.log(amount + ' dollars... no wait sorry...');
+  }
+}
+
+//6. Create a `verbing` code block. It should take a single string variable. If its length is at least 3, it should add 'ing' to its end, unless it already ends in 'ing', in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged.
+console.log(' ');
+console.log('Question 6~~~~~~~~~');
+console.log(' ');
+
+let verbWord = 'surprising';
+
+switch (verbWord.length >= 3) {
+  case true:
+    if (verbWord.slice(verbWord.length - 3) === 'ing') {
+      console.log(verbWord + 'ly');
+    } else {
+      console.log(verbWord + 'ing');
+    }
+
+    break;
+  default:
+    console.log(verbWord);
+}
+
+//7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
+console.log(' ');
+console.log('Question 7~~~~~~~~~');
+console.log(' ');
+
+let fullName = 'Boa Hancock';
+let withoutLast = '';
+
+for (i = fullName.length - 1; i >= 0; i--) {
```

Comment:
for (i = fullName.length - 1; i >= 0; i--) => Good practice would be to put let in front of i = fullName.length -1. Evne though this loop still works, it is not a good practice to do this.